www/pages/error/ErrorPage.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import Link from 'next/link';
+
+import styles from './ErrorPage.module.css';
+
+const ErrorPage = ({ statusCode }) => (
+    <div className={ styles.error }>
+        <h1>
+            {statusCode === 404 ?
+                <FormattedMessage id="error.not_found.title" /> :
+                <FormattedMessage id="error.internal.title" />
+            }
+        </h1>
+
+        <p>
+            <Link href="/">
+                <a>
+                    <FormattedMessage id="error.return_to_home" />
+                </a>
+            </Link>
+        </p>
+    </div>
+);
+
+ErrorPage.propTypes = {
+    statusCode: PropTypes.number.isRequired,
+    err: PropTypes.shape({
+        name: PropTypes.string.isRequired,
+        message: PropTypes.string.isRequired,
+        statusCode: PropTypes.number.isRequired,
+    }),
+};
+
+ErrorPage.getInitialProps = ({ res, err }) => {
+    let statusCode = 404;
+
```

Comment:
Please remove the ifs below if you accept my suggestion

Reply:
Accepted, wasn't fully aware that we had support for nullish coalescing / safe navigation.